src/main/java/com/minecolonies/coremod/commands/AbstractSingleCommand.java

```diff
@@ -57,4 +67,59 @@ public String getCommandUsage(@NotNull final ICommandSender sender)
         }
         return sb.toString();
     }
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)